pkg/kubecost/asset.go

```diff
@@ -2304,11 +2310,12 @@ func (sa *SharedAsset) String() string {
 // a window. An AssetSet is mutable, so treat it like a threadsafe map.
 type AssetSet struct {
 	sync.RWMutex
-	assets   map[string]Asset
-	props    []AssetProperty
-	Window   Window
-	Warnings []string
-	Errors   []string
+	aggStrings []string
+	assets     map[string]Asset
+	props      []AssetProperty
```

Comment:
Works for me! I didn't see any other purpose it served, but kept it around in case that was metadata we'd want on a given AssetSet. We can always put it back if it becomes relevant :)